docs/source/developers/continuous_integration/crossbow.rst

```diff
@@ -80,8 +80,8 @@ to step 3:
 
 1. `Create the queue repository`_
 
-2. Enable `TravisCI`_, `Appveyor`_, `Azure Pipelines`_ and `CircleCI`_
-   integrations on for the newly created queue repository.
+2. Enable `TravisCI`_, `Appveyor`_, and `Azure Pipelines`_ integrations on for
```

Comment:
I think we need to add 
Will need to add a link and enumeration item about Github Actions.

```suggestion
2. Enable `Github Actions`_, `Travis-CI`_ and `Azure Pipelines`_ integrations on for
```

Reply:
@pitrou, my understanding is that there are no explicit steps that a user needs to take to enable GitHub Actions in the queue repository, but I might be missing a step.

Reply:
Ah, cool, thank you.